bin/metadata.js

```diff
@@ -25,7 +25,7 @@ const MetadataGenerator = require('../lib/metadata_gen');
 const OWNER = 'nodejs';
 const REPO = 'node';
 
-const PR_ID = parseInt(process.argv[2]);  // example
+const PR_ID = parsePRId(process.argv[2]);  // example
```

Comment:
I think the `// example` comment can be removed now XD I added it when I chose 14782 as the default PR to test...I can do that later.

Reply:
done! :)